.coveragerc

```diff
@@ -1,5 +1,14 @@
+[run]
+source=evalml
+
 [report]
 exclude_lines =
-    # Have to re-enable the standard pragma
     pragma: no cover
-
+    def __repr__
+    raise AssertionError
+    raise NotImplementedError
+    if __name__ == .__main__.:
+    if self._verbose:
+    if verbose:
+    if profile:
+    pytest.skip
```

Comment:
@ParthivNaresh would you mind explaining why its necessary to add these lines? If they're not necessary, perhaps we can delete this change?

Reply:
I can absolutely get rid of it, I was trying to emulate the featuretools approach to the .coveragerc file as best as I could but I can delete them if they seem unnecessary in our case